JeremyM.txt

```diff
@@ -1,5 +1,6 @@
 Hello, I'm Jeremy Morales.
-I'd use quiet, observant, and calm to describe myself.
+I'd use loud, scattered, and gregarious to describe myself.
+
+I enjoy making furniture for small woodland creatures and scattering them around forested areas for them to use.
 
```

Comment:
I've never heard anyone do that before! Sweet gig -- Also add me as a reviewer so I can merge this repo 👍